src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  justify-content: left;
+  width: 706px;
+  padding: 10px;
+  flex-wrap: wrap;
+
+  & :nth-child(n) {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 100px;
+    min-height: 100px;
+    background-color: #eee;
```

Comment:
Sometimes you can add a blank line your CSS properties to make a cleaner code like in this case, but the order of it is fine

Reply:
Hi @jv-aquino, 

About this issue I can't make commit if my code don't pass in the coding-styles defined by mate-scripts(the blank line throws an error). Let me know if is possible to make this adjustments without got error in tests.

Reply:
If this is blocked by platform code-styling, there's no need for changing, just keep the advice in mind for future projects,